batch/batch/driver/main.py

```diff
@@ -273,13 +273,37 @@ async def deactivate_instance_1(instance):
     log.info(f'deactivating {instance}')
     await instance.deactivate('deactivated')
     await instance.mark_healthy()
-    return web.Response()
 
 
 @routes.post('/api/v1alpha/instances/deactivate')
 @active_instances_only
 async def deactivate_instance(request, instance):  # pylint: disable=unused-argument
-    return await asyncio.shield(deactivate_instance_1(instance))
+    await asyncio.shield(deactivate_instance_1(instance))
+    return web.Response()
+
+
+@routes.post('/instances/{instance_name}/deactivate')
+@check_csrf_token
+@web_authenticated_developers_only()
+async def kill_instance(request, userdata):
```

Comment:
You'll need a comment here like on line 280 to tell `pylint` that it's ok we're not using the userdata parameter.